modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -334,7 +335,7 @@ spec:
   configuration:
     velero:
       defaultPlugins:
-        - kubevirt <2>
+        - kubevirt <2> // correction required
```

Comment:
@amastbau && @weshayutin - following from [Slack message about an error in the docs](https://redhat-internal.slack.com/archives/C0144ECKUJ0/p1719230408853689?thread_ts=1719220386.143819&cid=C0144ECKUJ0)

Please can I ask for advice